scopes/api.py

```diff
@@ -52,11 +48,36 @@ def paginator(self):
             self._paginator = DefaultPagination()
         return self._paginator
 
-    @classmethod
-    def _create_oidc_scopes_data(cls):
-        if hasattr(cls, '_oidc_scopes_data'):
-            return cls._oidc_scopes_data
 
+class ScopeDataBuilder:
+    """
+    A builder for scope data to be used in the API.
+
+    Implemented as a class to provide better control of caching. A ScopeDataBuilder instance caches
+    ApiScopes (which are in the db), so in practice there probably should be an own instance per request
```

Comment:
"in practice there probably should be" ... Can you make this a bit more specific?